<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestStringListModel</class>
 <widget class="QMainWindow" name="TestStringListModel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1020</width>
    <height>750</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>750</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1020</width>
    <height>800</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Times New Roman</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>TestStringListModel</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>981</width>
      <height>601</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>QListView</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QPushButton" name="btnIniList">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>100</height>
           </size>
          </property>
          <property name="text">
           <string>初始化列表</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="btnListAppend">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>100</height>
           </size>
          </property>
          <property name="text">
           <string>添加项</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="btnListInsert">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>400</width>
            <height>100</height>
           </size>
          </property>
          <property name="text">
           <string>插入项</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="btnListDelete">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>400</width>
            <height>100</height>
           </size>
          </property>
          <property name="text">
           <string>删除当前项</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="btnListClear">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>400</width>
            <height>100</height>
           </size>
          </property>
          <property name="text">
           <string>清除列表</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListView" name="listView">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>160</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>20000</width>
          <height>20000</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>QPlainTextEdit</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="btnTextClear">
          <property name="minimumSize">
           <size>
            <width>250</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>2000</width>
            <height>100</height>
           </size>
          </property>
          <property name="text">
           <string>清空文本</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnTextImport">
          <property name="minimumSize">
           <size>
            <width>250</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>2000</width>
            <height>100</height>
           </size>
          </property>
          <property name="text">
           <string>显示数据模型的StringList</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="plainTextEdit"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLabel" name="LabInfo1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>630</y>
      <width>400</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>400</width>
      <height>50</height>
     </size>
    </property>
    <property name="text">
     <string>当前项的ModelIndex</string>
    </property>
   </widget>
   <widget class="QLabel" name="LabInfo">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>630</y>
      <width>400</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>400</width>
      <height>50</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1020</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
